dwave/plugins/torch/samplers/annealing.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2025 D-Wave
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+import torch
+
+from dwave.plugins.torch.models.boltzmann_machine import GraphRestrictedBoltzmannMachine
+from dwave.plugins.torch.samplers._base import TorchSampler
+from dwave.plugins.torch.utils import sampleset_to_tensor
+from hybrid.composers import AggregatedSamples
+
+if TYPE_CHECKING:
+    import dimod
+    from dwave.plugins.torch.models.boltzmann_machine import GraphRestrictedBoltzmannMachine
+
+
+spread = AggregatedSamples.spread
+
+
+__all__ = ["DimodSampler"]
+
+
+class DimodSampler(TorchSampler):
```

Comment:
Why is the module named `annealing` instead of, e.g., `dimod`? Many dimod samplers are not annealing-based, e.g., greedy and tree-decomposition-based sampler/solver.